src/App.js

```diff
@@ -0,0 +1,153 @@
+import React, { Component } from 'react';
+import './App.css';
+import Header from './components/Header.js';
+import Add from './components/Add.js';
+import Delete from './components/Delete.js';
+import Save from './components/Save.js'
+import Titles from './components/Titles.js';
+import Note from './components/Note.js';
+
+class App extends Component {
+	state = {
+		currentNote: {
+			id: null,
+			title: "",
+			text: ""
+		},
+		allNotes: {
+			1: {
+				id: 1,
+				title: "Sometext",
+				text: "here is some text"
+			},
+			2: {
+				id: 2,
+				title: "Moretext",
+				text: "here is more text"
+			}
+		}
+	}
+
+	componentDidMount() {
+		fetch('https://noteworthy-graphql.herokuapp.com/v1/graphql', {
```

Comment:
need to add a `.catch()` to you fetch

Reply:
you might want to put your DB querying in a separate file and import it, and call it.